test/rekt/resources/configmap/broker/broker_config.yaml

```diff
@@ -30,3 +30,11 @@ data:
   {{ if .authSecret }}
   auth.secret.ref.name: "{{ .authSecret }}"
   {{ end }}
+  {{ if .kafkaChannel }}
+  channel-template-spec: |
```

Comment:
in theory if we stash `KafkaChannel` here, we do not need the other values - but even if they are there, they are not causing any harm.  Just a note

Reply:
> we do not need the other values
What do you mean by "other values"? Thanks